src/mongocxx/test/database.cpp

```diff
@@ -38,6 +38,21 @@ TEST_CASE("A default constructed database is false-ish", "[database]") {
     REQUIRE(!d);
 }
 
+TEST_CASE("database copy", "[database]") {
+    client mongodb_client{uri{}};
+
+    std::string dbname{"foo"};
+    std::string dbname2{"bar"};
+    database db = mongodb_client[dbname];
+
+    database db2 = db;
+    database db3 = mongodb_client[dbname2];
+    db3 = db;
+
```

Comment:
it looks like you are testing the copy assignment operator, but not the copy constructor. Can you test the copy constructor as well?